lib/phoenix/live_dashboard/pages/os_mon_page.ex

```diff
@@ -188,11 +188,11 @@ defmodule Phoenix.LiveDashboard.OSMonPage do
                 <div phx-hook="PhxColorBarHighlight" id="cpu-color-bars">
                   <%= for {num_cpu, usage} <- @os_mon.cpu_per_core do %>
                     <div class="progress flex-grow-1 mb-3">
-                      <%= live_component @socket, ColorBarComponent, id: {:cpu, num_cpu}, data: cpu_usage_sections(usage), title: "CPU #{num_cpu+1}" %>
```

Comment:
@josevalim This was one that was stateful. It workes great without an `id` as well.